Package.swift

```diff
@@ -4,7 +4,8 @@ import PackageDescription
 let package = Package(
     name: "swift-bson",
     platforms: [
-        .macOS(.v10_14)
+        .macOS(.v10_14),
+        .iOS(.v11)
```

Comment:
we use some Swift APIs that are only available on iOS 11+, so this is the lowest possible value of this. But per the scope I need to do a little research to figure out what a reasonable choice for this is before we release.